test/unit/libraries/k8s_config_map_test.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'resource_test'
+
+class K8sConfigMapTest < ResourceTest
+  STUB_DATA = {
+    'v1': {
+      default: {
+        configmaps: [
+          {
+            name: 'configmap1',
+            kind: 'configmap',
+            status: {
+              phase: 'running'
+            },
+            metadata: {
+              uid: 'abcd1234',
+              name: 'configmap1',
+              namespace: 'default',
+              resourceVersion: 1234,
+              annotations: [],
```

Comment:
annotations and labels is not a Array if I am not wrong and may need to treat it as hash.